Source/Serilog.Enrichers.Span/SpanOptions.cs

```diff
@@ -9,4 +9,9 @@ public class SpanOptions
     /// Gets or sets a value indicating whether to include tags in log or not. Default is false.
     /// </summary>
     public bool IncludeTags { get; set; }
+
+    /// <summary>
+    /// Gets or sets log properties names for span.
+    /// </summary>
+    public SpanLogEventPropertiesNames LogEventPropertiesNames { get; set; } = new();
```

Comment:
This can be null. We'd need to protect against that. Alternatively, we can just add the property names directly into `SpanOptions` since there are only three.